src/Microsoft.ML.TimeSeries/DTRootCauseLocalization.cs

```diff
@@ -0,0 +1,306 @@
+                return TryUnparseCore(sb);
+            }
+        }
+
+        internal class Options : TransformInputBase
+        {
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "New column definition(s) (optional form: name:src)", Name = "Column", ShortName = "col", SortOrder = 1)]
+            public Column[] Columns;
+
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Weight for getting the score for the root cause item.", ShortName = "Beta", SortOrder = 2)]
+            public double Beta = DTRootCauseLocalizationEstimator.Defaults.Beta;
+
+        }
+
+        /// <summary>
+        /// The input and output column pairs passed to this <see cref="ITransformer"/>.
+        /// </summary>
+        internal IReadOnlyCollection<(string outputColumnName, string inputColumnName)> Columns => ColumnPairs.AsReadOnly();
+
+        private readonly double _beta;
+
+        /// <summary>
+        /// Localization root cause for multi-dimensional anomaly.
+        /// </summary>
+        /// <param name="env">The estimator's local <see cref="IHostEnvironment"/>.</param>
+        /// <param name="beta">Weight for generating score.</param>
+        /// <param name="columns">The name of the columns (first item of the tuple), and the name of the resulting output column (second item of the tuple).</param>
+
+        internal DTRootCauseLocalizationTransformer(IHostEnvironment env, double beta = DTRootCauseLocalizationEstimator.Defaults.Beta, params (string outputColumnName, string inputColumnName)[] columns)
+            : base(Contracts.CheckRef(env, nameof(env)).Register(RegistrationName), columns)
+        {
+            Host.CheckUserArg(beta >= 0 && beta <= 1, nameof(Options.Beta), "Must be in [0,1]");
+
+            _beta = beta;
+        }
+
+        // Factory method for SignatureDataTransform.
+        internal static IDataTransform Create(IHostEnvironment env, Options options, IDataView input)
+        {
+            Contracts.CheckValue(env, nameof(env));
+            env.CheckValue(options, nameof(options));
+            env.CheckValue(input, nameof(input));
+            env.CheckValue(options.Columns, nameof(options.Columns));
+
+            return new DTRootCauseLocalizationTransformer(env, options.Beta, options.Columns.Select(x => (x.Name, x.Source ?? x.Name)).ToArray())
+                .MakeDataTransform(input);
+        }
+
+        // Factory method for SignatureLoadModel.
+        private static DTRootCauseLocalizationTransformer Create(IHostEnvironment env, ModelLoadContext ctx)
+        {
+            Contracts.CheckValue(env, nameof(env));
+            var host = env.Register(RegistrationName);
+            host.CheckValue(ctx, nameof(ctx));
+            ctx.CheckAtModel(GetVersionInfo());
+            return new DTRootCauseLocalizationTransformer(host, ctx);
+        }
+
+        private DTRootCauseLocalizationTransformer(IHost host, ModelLoadContext ctx)
```

Comment:
>DTRootCauseLocalizationTransformer [](start = 16, length = 34)

This constructor loads the transformer from a serialized model, so you need to read `_beta` here.

Reply:
> > DTRootCauseLocalizationTransformer [](start = 16, length = 34)
> 
> This constructor loads the transformer from a serialized model, so you need to read `_beta` here.

Updated